portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,36 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Rishith Kyatham Personal Portfolio</h1>
+      <p>I am a Sophomore in Computer Science double Majoring in Computer Science and Applied Mathematics & Science. My interest leans towards AI and Machine Learning and I am aiming to become an aspiring Full-Stack Developer in the future.</p>
```

Comment:
nit: this seems like it's over 80 characters. Try to keep the lines length under 80 characters per https://developers.google.com/style/html-formatting#line-length

Reply:
Done